### PR TITLE
build: added figma-tokens parse pipeline

### PR DESCRIPTION
**Pull request by SavelevMatthew**

### CODE REVIEW

File: packages/ui/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  "repository": "https://github.com/open-condo-software/condo/blob/master/packages/ui",
+  "author": {
+    "name": "Matthew Savelev",
+    "email": "msavelyev@doma.ai",
+    "url": "https://github.com/SavelevMatthew"
+  },
+  "license": "MIT",
+  "scripts": {
+    "prepare:tokens": "rm -rf src/tokens/sets && node bin/prepare-tokens.js",
+    "build:tokens": "find src/tokens -name 'variables.*' -delete && node bin/build-vars.js",
```
Comment (pahaz) [inline packages/ui/package.json:14]: why do you need to delete `bin/build-vars.js`?

Comment (pahaz) [inline packages/ui/package.json:14]: it looks like a part of `prepare:tokens` command

File: packages/ui/sd.config.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+const path = require('path')
+
+module.exports = {
+    source: [path.join(__dirname, 'src/tokens/sets/*.json')],
+    platforms: {
```
Comment (pahaz) [inline packages/ui/sd.config.js:5]: What is the `sd.config` file?

File: packages/ui/package.json (hunk 1 above)
Comment (SavelevMatthew) [reply packages/ui/package.json:14]: Sometimes we don't want to generate tokens all the times. But maybe you're right...

File: packages/ui/sd.config.js (hunk 2 above)
Comment (SavelevMatthew) [reply packages/ui/sd.config.js:5]: config for `style-dictionary`

File: packages/ui/bin/prepare-tokens.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+                }, null, 2))
+                console.log(`✓ Set ${setName} is processed`)
+            } else {
+                throw Error(`✖ Error occurred while processing ${setName} set`)
+            }
+        })
+    }
+}
+
+prepareTokens(...process.argv.slice(2))
```
Comment (sitozzz) [inline packages/ui/bin/prepare-tokens.js:61]: Check for args -> throw readable error

**Review by sitozzz — APPROVED** [review]

File: packages/ui/bin/prepare-tokens.js (hunk 3 above)
Comment (SavelevMatthew) [reply packages/ui/bin/prepare-tokens.js:61]: There's only 1 arg...and it's not necessary...So I guess it will be redundant check

**Pull request merged**